src/Rules/Resources/Descriptions.resx

```diff
@@ -119,375 +119,375 @@
+    <comment>The description of an accessibility problem with an element currently under inspection. The term "ItemType" should not be translated (i.e. it should be treated as a loanword). {Locked="ItemType"}</comment>
   </data>
   <data name="NameEmptyButElementNotKeyboardFocusable" xml:space="preserve">
     <value>The Name property for the given element is empty, but the element isn't focusable. Please consider whether or not the element should have a name.</value>
-    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" (with uppercase N) should not be translated (i.e. it should be treated as a loanword).</comment>
+    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" (with uppercase N) should not be translated (i.e. it should be treated as a loanword). {Locked="Name"}</comment>
   </data>
   <data name="NameNoSiblingsOfSameType" xml:space="preserve">
     <value>The Name property of the given element may be null or empty if the element has no siblings of the same type.</value>
-    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" should not be translated (i.e. it should be treated as a loanword). {Locked="Name"}</comment>
   </data>
   <data name="NameNullButElementNotKeyboardFocusable" xml:space="preserve">
     <value>The Name property for the given element is null, but the element isn't focusable. Please consider whether or not the element should have a name.</value>
-    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" (with uppercase N) should not be translated (i.e. it should be treated as a loanword).</comment>
+    <comment>The description of an accessibility problem with an element currently under inspection. The term "Name" (with uppercase N) should not be translated (i.e. it should be treated as a loanword). {Locked="Name"}</comment>
   </data>
   <data name="NameOnCustomWithParentWPFDataItem" xml:space="preserve">
     <value>The Name property of a custom control may be empty if the parent is a WPF dataitem.</value>
-    <comment>The description of an accessibility problem with an element currently under inspection. The terms "Name" and "dataitem" should not be translated (i.e. they should be treated as loanwords).</comment>
+    <comment>The description of an accessibility problem with an element currently under inspection. The terms "Name" and "dataitem" should not be translated (i.e. they should be treated as loanwords). {Locked="Name", "DataItem"}</comment>
```

Comment:
In the original comment, "dataitem" was all lower case, but it's PascalCase "DataItem" in the locked. I'm guessing that the original spelling was a typo ?